.github/workflows/nv-sd.yml

```diff
@@ -0,0 +1,54 @@
+name: nv-a6000
+
+on:
+  schedule:
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  unit-tests:
+    runs-on: [self-hosted, nvidia, a6000]
```

Comment:
Not sure what hardware we want this running on, but specify here?

Reply:
For stable diffusion we want to target A6000 due to Triton's architecture support.